project/frontend/src/components/ResultsMap.js

```diff
@@ -3,39 +3,54 @@ import React, { useState } from 'react';
 
 const mapStyle = {
   border: 'thin solid black',
-  height: '500px',
-  width: '500px',
+  height: '100%',
+  width: '100%',
 };
 
 function MapContainer(props) {
-  const [activeMarker, changeMarker] = useState({});
-  const [showInfoWindow, changeInfoVisibility] = useState(false);
-
-  const coords = { lat: 40.837, lng: -73.865 };
+  const [activeMarker, setActiveMarker] = useState({});
+  const [showInfoWindow, setShowInfoWindow] = useState(false);
+  const restaurants = props.restaurants;
+  const userCenter = props.userLocation;
 
   const onMarkerClick = (props, marker) => {
```

Comment:
props seems unused - can we remove it?

Reply:
for some reason when I remove props here i get an error!

Reply:
<img width="1133" alt="Screen Shot 2020-07-16 at 1 41 30 PM" src="https://user-images.githubusercontent.com/43529415/87704258-233ecb00-c76a-11ea-91a0-9d07ee1ea668.png">
^if i remove props

Reply:
hmm kk, seems like its trying to use something out of props to handle errors internally (just skimming through the doc linked in the error) but thanks for looking!